css/index.css

```diff
@@ -143,4 +257,47 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+.aboutp {
+    margin-top: 20px;
+    padding: 20px;
+    border-bottom: dotted 1px black;
```

Comment:
So here it would've been better to do 1px dashed black. If you're ever not seeing the match with the design file, go look at the documentation for css https://www.w3schools.com/css/css_border.asp